content/blog/2023-06-21-comparison-of-ethereum-classic-and-fiat-monetary-policies/index.md

```diff
@@ -0,0 +1,146 @@
+
+## Eras
+
+Ethereum Classic’s issuance is divided in eras. Fiat money issuance is unpredictable and decided arbitrarily by a committee whenever they wish.
+
+Eras in ETC are measured in blocks in which the payments to miners, thus the issuance of coins, will be the same for each era.
+
+Ethereum Classic’s eras last 5 million blocks which, at 13 seconds per block, turns out to be more or less 2 years per era.
+
+After each era ETC discounts the payment to miners.
+
+## Discount Per Eras
+
+In Ethereum Classic, the discount per era is 20%. 
+This means that since genesis, ETC has paid miners 5 ETC per block in the first era, then 4 ETC, then 3.20, and now is paying 2.56 which is the current era.
+
+Fiat money issuance has no eras, is decided arbitrarily by the government, and there are no parameters or limits to their actions.
+
+This results in patterns as the one seen in the Federal Reserve’s assets (which is analogous to the monetary base) chart above. There is no certainty as to the money supply, it is prone to abuse and over emission, creating chronic inflation, and letting the government loose to overspend, leading nations to bankruptcy and poverty.
+
+## Pre-Mine
+
+Because Ethereum Classic and Ethereum were one single project since the beginning, and the project started with a crowd sale that resulted in an initial issuance, or what is called a “pre-mine”, then both ETC and ETH have a pre-mine stock of coins that were created at genesis block.
+
+This premine stock for Ethereum Classic was 72,000,000 ETC of the current supply of 140,600,000.
+
+Fiat money has an infinite pre-mine. This is because, as many central bankers have famously explained on national television, they can just go to a computer terminal and add trillions of units of the currency to the economy whenever they wish.
+
+This means that central banks have an infinite stock of the currency, it’s just their prerogative how much they will use of it at any time.
+
+## Issuance
+
+The current era in ETC is era 4 which is between block 15,000,001 and 20,000,000. This means it is paying 2.56 ETC per block to miners as explained before.
+
+The next payments will be discounted by 20% per era lowering them to 2.048 ETC, then 1.6384, 1.311072, and so on until it diminishes completely approximately by the year 2140.
+
+As said several times before in this post, central banks may issue currency whenever they wish. There is no program, certainty, or limitations whatsoever other than the government officials’ opinions and biases.
+
+## Coin Burning
+
+Fiat money has a coin burning mechanism and ETC does not.
+
+The way the coin burning works in fiat money may be explained by describing how it is issued first. To issue money, central bankers may do two thing, a) credit newly issued money in the accounts of banks directly when they require loans, and b) buy government bonds to finance the state, or buy other securities in the open market, all with newly issued currency.
```

Comment:
Line 120: Remove unneeded article; Change noun to plural; Correct preposition usage
```suggestion
The way coin burning works in fiat money may be explained by describing how it is issued first. To issue money, central bankers may do two things, a) credit newly issued money to the accounts of banks directly when they require loans, and b) buy government bonds to finance the state, or buy other securities in the open market, all with newly issued currency.
```